src/pages/development/build/component-file-structure.md

```diff
@@ -23,12 +23,22 @@ A typical file structure for a module can look like the following:
 
 Following are some common module directories:
 
-*  `Block`: contains PHP view classes as part of Model View Controller(MVC) vertical implementation of module logic.
-*  `Controller`: contains PHP controller classes as part of MVC vertical implementation of module logic.
-*  `etc`: contains configuration files; in particular, `module.xml`, which is required.
-*  `Model`: contains PHP model classes as part of MVC vertical implementation of module logic.
-*  `Setup`: contains classes for module database structure and data setup which are invoked when installing or upgrading.
-*  `ViewModel`: contains PHP model classes as part of a model-view-viewmodel (MVVM) implementation. It allows developers to offload features and business logic from block classes into separate classes that are easier to maintain, test, and reuse.
+* `Block`: contains PHP view classes as part of Model View Controller(MVC) vertical implementation of module logic.
+* `Controller`: contains PHP controller classes as part of MVC vertical implementation of module logic.
+* `etc`: contains configuration files; in particular, `module.xml`, which is required.
+* `Model/ResourceModel/Colletions`: contains PHP model classes as part of MVC vertical implementation of module logic.
+* `Setup`: contains classes for module database structure and data setup which are invoked when installing or upgrading.
+* `ViewModel`: contains PHP model classes as part of a model-view-viewmodel (MVVM) implementation. It allows developers to offload features and business logic from block classes into separate classes that are easier to maintain, test, and reuse.
+
+### Model file structure
+
+The directory structure for the ORM follows the MVC pattern:
+
+```tree
```

Comment:
I think it would be easier to read if the directories were defined in a list below the tree. In the current state, one must scroll to read.

Reply:
The idea here is to show the hierarchy, this information is useful and appears on the exam

Reply:
Yes, the tree is fine. I meant the definitions of the items in the tree. The definitions distort the tree and cause scrolling. Code blocks don't wrap long lines. Moving the definitions to a list below could help visually.

Reply:
But I don't see a problem, because the information didn't exist.